src/pages/Authentication/Singup.jsx

```diff
@@ -1,18 +1,20 @@
 import React from "react";
 import { Link } from "react-router-dom";
 import { Footer, Navbar } from "../../components";
+import { useloginsignup } from "../../hooks/context/loginSignup-context";
 import "./Authentication.css";
 
 export const Singup = () => {
+  const {showpassword,showpasswordFun}=useloginsignup();
```

Comment:
You can use a proper naming convention for more readability of code. Eg: showPasswordFun (Apply this in all)